core/tests/protractor_utils/AdminPage.js

```diff
@@ -272,6 +287,50 @@ var AdminPage = function() {
       expect(name).toEqual(expectedUsernamesArray[j]);
     }
   };
+
+  this.addReviewer = async function(name, newRole) {
+    await waitFor.elementToBeClickable(
+      adminRolesTab, 'Admin Roles tab is not clickable');
+    await adminRolesTab.click();
+
+    // Change values for "add reviewer for community" form, and submit it.
+    await waitFor.visibilityOf(
+      addReviewerName, 'Add reviewer name is not visible');
+    await addReviewerName.sendKeys(name);
+
+    var roleOption = selectReviewerRole.element(
+      by.cssContainingText('option', newRole));
+    await waitFor.visibilityOf(
+      roleOption, 'Reviewer role option is not visible');
```

Comment:
Can you check for clickability instead, or even better, use our new actions utilities: https://github.com/oppia/oppia/blob/develop/core/tests/protractor_utils/action.js

Reply:
Done!